tools/mo/requirements.txt

```diff
@@ -1,9 +1,9 @@
-tensorflow>=1.15.5,<2.6
+tensorflow>=1.15.5
 mxnet~=1.2.0; sys_platform == 'win32'
 mxnet~=1.7.0.post2; sys_platform != 'win32'
 networkx~=2.5; python_version <= "3.6"
 networkx<2.8.1; python_version > "3.6"
-numpy>=1.16.6,<1.20
+numpy>=1.16.6
```

Comment:
Is it "just" possible now? As far as I remember there was a reason to restrict version of `numpy`.

Reply:
So if you want to relax it as much as possible, I assume that you need to test it out other way. Right now CI machines will install "latest possible" version for each system-python configuration.

Reply:
@jiwaszki, I expect from the validation team to use separate requirements file with fixed dependecies versions so we avoid non-deterministic results. So before merging this we need some change in the validation system

Reply:
Has validation team been notified about new testing requirements? Is there an ETA? @rkazants

Reply:
@p-wysocki, yes, they are notified. ETA is not yet defined.